api/handler/acl.go

```diff
@@ -56,13 +56,17 @@ const (
 	s3GetObjectVersion           = "s3:GetObjectVersion"
 )
 
-// AWSACL is aws permission constants.
-type AWSACL string
```

Comment:
Doesn't seem to be very useful.

Reply:
It's not AWS ACL, it's permissions. Also it is unused outside the package, and has couple of unexported constants. I wanna realize all these things from the definition.

Reply:
Btw all values are defined in https://pkg.go.dev/github.com/aws/aws-sdk-go/service/s3#pkg-constants. I don't know why we define them locally.

Reply:
I guess the main problem is that this package has _a lot_ more inside.